src/EditorFeatures/Core/Editor/EditorLayerExtensionManager.cs

```diff
@@ -55,7 +55,9 @@ internal class ExtensionManager(
 
             public override void HandleException(object provider, Exception exception)
             {
+#pragma warning disable FixAllRefactoring // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
                 if (provider is CodeFixProvider or CodeFixes.FixAllProvider or CodeRefactoringProvider or CodeRefactorings.FixAllProvider)
+#pragma warning restore FixAllRefactoring // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
```

Comment:
📝 I'd like to implement a diagnostic suppressor in the package added by #69469 which removes the need to disable experimental API warnings for features we define in locations where they do not impact public APIs.